numba/tests/test_unicode.py

```diff
@@ -1171,6 +1176,54 @@ def test_istitle(self):
             self.assertEqual(py_result, c_result,
                              error_msg.format(s, py_result, c_result))
 
+    def test_translate(self):
+        pyfunc = translate_usecase
+        cfunc = njit(pyfunc)
+
+        # Samples taken from CPython testing:
+        # https://github.com/python/cpython/blob/865c3b257fe38154a4320c7ee6afb416f665b9c2/Lib/test/test_unicode.py#L313-L381     # noqa: E501
+        cpython = ['bbbc', 'abababc', 'iiic', 'iiix', 'c', 'xzx']
+        extra = ['处', '[a]', '[a\xe9]', '\u20aa', '\u20ac']
+        table = [
+            {ord('a'):ord('c'), ord('b') : ord('n')},
+            {ord('a'): ord(' '), ord('b'): ord('i'), ord('c'): ord('x')},
+        ]
+
+        table_extra = [
+            {ord('a'): ord('\x04'), ord('\xe9'): ord('t')},
+            {ord('\u20aa'): ord('\u20ac')}
+        ]
+
+        msg = 'Results "{}".translate({}) must be equal'
+        for s in UNICODE_EXAMPLES + cpython + extra:
+            for t in table + table_extra:
+                numba_t = numba.typed.Dict()
+                for key, value in t.items():
+                    numba_t[key] = value
+                self.assertEqual(pyfunc(s, t), cfunc(s, numba_t),
+                                 msg=msg.format(s, t))
+
+    @unittest.skip('Dont work in numba')
```

Comment:
This needs resolving, what's the issue?

Reply:
https://github.com/numba/numba/issues/4873